src/templates/css/styles.css

```diff
@@ -5,6 +5,7 @@ figure {
 /* layout */
 .body {
   background: #fff;
+  font-family: Roboto,sans-serif;
```

Comment:
I was following this [template](https://www.getmdl.io/templates/article/index.html) which doesn't use Roboto for the text (only for headers). Otherwise LGTM. If we do this we should use `'Roboto','Helvetica','Arial',sans-serif;` Can you also check whether this changes the alignment between the lines in the code and doc sections?